repository/e.json

```diff
@@ -780,6 +780,17 @@
 				}
 			]
 		},
+		{
+			"name": "Ember & Glimmer Syntax",
```

Comment:
You can simply name the package after the language itself, "Ember & Glimmer" and leave out the "syntax" because syntaxes do quite a bit more than highlighting in Sublime Text. 

If you add `"previous names": ["Ember Syntax"]`, and remove their entry lower in this file. That way we can replace the existing package with yours and upgrade any existing users to your more up to date implementation.

Is your syntax a fork of handlebars in some way? How does it relate? Because your syntax still presents itself as a handlebars syntax, which might confuse people and other packages if that's not correct.